site/docs/components/static-list/examples.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+  <LivePreview componentName="static-list" exampleName="DefaultSingle" >
+
+## Default Single
+
+Default `Static list` with a single item.
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="DefaultMultiple" >
+
+## Default Multiple
+
+Default `Static list` group showing multiple items stacked together. 
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="AdditionalLabel" >
+
+  ## Additional label
+
+`Static list` items with additional label  grouped together to organize related content.
```

Comment:
Any potential design guide around choice of secondary label's `color` ?

Reply:
No so far, the recommendation was to keep the component simple and treat color at the pattern level. For instance when used in file upload, error validation will be required error-decorative/informative. Do you think it is worthy to add an example at the component level?